### PR TITLE
Update Symfony components after 6.4.7 release

### PR DESCRIPTION
**Pull request by nicosomb**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Update Symfony components after 6.4.7 release
| Type?             | improvement 
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | 
| UI Tests          | https://github.com/nicosomb/ga.tests.ui.pr/actions/runs/8879344036
| Sponsor company   | PrestaShop SA

### CODE REVIEW

**Review by jolelievre — DISMISSED** [review]
Thanks @nicosomb

**Review by kpodemski — DISMISSED** [review]

**Comment by nicosomb** [comment]
@Julievrz I had to change a message in a file. I don't know the correct process for that. Tell me if I have to do something please.

File: src/PrestaShopBundle/Form/Admin/Sell/Customer/CustomerType.php [hunk 1]
```diff
@@ -364,12 +364,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
                             'min' => 0,
                             'max' => Validate::MYSQL_UNSIGNED_INT_MAX,
-                            'minMessage' => $this->trans(
-                                '%s is invalid. Please enter an integer greater than or equal to 0.',
-                                'Admin.Notifications.Error'
-                            ),
-                            'maxMessage' => $this->trans(
-                                '%s is invalid. Please enter an integer lower than or equal to %s.',
+                            'notInRangeMessage' => $this->trans(
+                                '%s is invalid. Please enter an integer greater than or equal to 0 and lower than or equal to %s.',
```
Comment (Julievrz) [inline src/PrestaShopBundle/Form/Admin/Sell/Customer/CustomerType.php]: Couldn't we say "%s is invalid. Please enter an integer between 0 and %s."?

Comment (nicosomb) [reply src/PrestaShopBundle/Form/Admin/Sell/Customer/CustomerType.php]: Sure we can, it will be simpler.

**Comment by nicosomb** [comment]
UI tests are green: https://github.com/nicosomb/ga.tests.ui.pr/actions/runs/8879344036

**Review by matthieu-rolland — APPROVED** [review]

**Review by tleon — APPROVED** [review]

**Pull request merged**